instrumentation/jax-rs-3.0/build.gradle

```diff
@@ -0,0 +1,36 @@
+dependencies {
+    implementation(project(":agent-bridge"))
+    implementation("jakarta.ws.rs:jakarta.ws.rs-api:3.1.0")
+    implementation ('org.glassfish.jersey.inject:jersey-hk2:3.0.5')
+
+    testImplementation('org.glassfish.jersey.containers:jersey-container-servlet:3.0.5')
+    testImplementation('org.glassfish.jersey.test-framework:jersey-test-framework-core:3.0.5')
+    testImplementation('org.glassfish.jersey.test-framework.providers:jersey-test-framework-provider-grizzly2:3.0.5')
+    testImplementation(project(":instrumentation:jersey-3"))
+}
+
+jar {
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.jax-rs-3.0',
+    'Implementation-Title-Alias': 'rest_annotations'}
+}
+
+verifyInstrumentation {
+    passesOnly 'jakarta.ws.rs:jakarta.ws.rs-api:[3.0-M1,)'
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
```

Comment:
Will this work in a Jakarta 9 app running with Java 8?

Reply:
ok, done